R/getPlot.R

```diff
@@ -0,0 +1,133 @@
+#' Get a plot
+#'
+#' Gets a plot for an FcsFile. If in RStudio or Jupyter and `raw`
+#' is not `TRUE`, displays the image. Otherwise, returns the raw,
+#' PNG-encoded bytes.
+#'
+#' @param experimentId character ID of experiment.
+#' @param fcsFileId character ID of FcsFile
+#' @param plotType character Plot type, one of: "contour", "dot", "density", or "histogram"
+#' @param xChannel character X channel name
+#' @param yChannel character Y channel name
+#' @param zChannel character Z channel name
+#' @param params list Optional query parameters:
+#' - axesQ logical Display axes lines. Defaults to TRUE.
+#' - axisLabelsQ logical Display axis labels. Defaults to TRUE.
+#' - color character Case-insensitive string in the format #rgb, #rgba,
+#'   #rrggbb or #rrggbbaa. The foreground color, i.e. color of curves in
+#'   "histogram" and "contour" plots, dots in "dot" plots.
+#' - compensation character ID of compensation to use for gating and
+#'   display.
+#' - gateLabel character One of "eventcount", "mean", "median", "percent",
+#'   "mad", "cv", "stddev", "geometricmean", "name", "none".
+#' - gateLabelFontSize double Font size for gate labels.
+#' - height double Image height. Defaults to 228.
+#' - percentileStart double For contour plots, the percentile of the first
+#'   contour.
+#' - percentileStep double For contour plots, the percentile step between
+#'   each contour.
+#' - populationId character Population ID. Defaults to ungated.
+#' - postSubsampleN double Randomly subsample the file to contain this many
+#'   events after gating.
+#' - postSubsampleP double Randomly subsample the file to contain this
+#'   percent of events (0 to 1) after gating.
+#' - renderGates logical Render gates into the image.
+#' - seed double Seed for random number generator used for subsampling. Use
+#'   for deterministic reproducible subsampling. If omitted, a pseudo-random
+#'   value is used.
+#' - smoothing double For density and contour plots, adjusts the amount of
+#'   smoothing. Defaults to 0 (no smoothing). Set to 1 for typical smoothing.
+#'   Higher values (up to 10) increase smoothing.
+#' - strokeThickness double The thickness of histogram and contour plot
+#'   lines. Defaults to 1.
+#' - tickLabelsQ logical Display tick labels. Defaults to false.
+#' - ticksQ logical Display ticks. Defaults to true.
```

Comment:
Nit: can you go through and make these all `TRUE`/`FALSE`? (Cf. lines 14, 15, 58).

Reply:
:+1: